github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -416,6 +416,7 @@ presubmits:
         - /usr/local/bin/runner.sh
         - /bin/sh
         - -c
+        - export SLIM=true &&
```

Comment:
Could you add it as an env var to the job config instead? Like here - https://github.com/kubevirt/project-infra/pull/3688/files#diff-ab108ff0d13755bd671fe2f0a2397f0c0290dc6655d2a03659b2cee3fa5958f7L472

Reply:
Sure, done.